src/index.html

```diff
@@ -22,6 +22,399 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
```

Comment:
The `href` attribute in the logo link should lead to the root of the website, usually `/` or `/#`, not `index.html`. This ensures that the link remains valid regardless of the current URL structure.